openquake/hazardlib/gsim/chiou_youngs_2008_swiss.py

```diff
@@ -56,8 +57,11 @@ class ChiouYoungs2008SWISS01(ChiouYoungs2008):
 
     def get_mean_and_stddevs(self, sites, rup, dists, imt, stddev_types):
 
+        ref_sites = copy.deepcopy(sites)
+        ref_sites.vs30 = 620 * np.ones(len(sites.vs30))
```

Comment:
So here it seems that you are changing the vs30 to the reference value of 600 m/s, but z1.0 remains unchanged from the values provided with the original vs30 values. Is this correct?